change/@ni-nimble-components-8cad4d26-536b-4f4b-890f-233d13b36bd0.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Update box shadow tokens, including removing the 'popupBoxShadowColor' token",
+  "packageName": "@ni/nimble-components",
+  "email": "20542556+mollykreis@users.noreply.github.com",
+  "dependentChangeType": "patch"
```

I'm not sure if we've defined a policy for this, but what do you think changing this so that that nimble-angular also gets a major bump? Since clients of nimble-angular who depend on the token will be broken, I think this makes sense.
```suggestion
  "dependentChangeType": "major"
```